test/scalability/default_generator_config.yaml

```diff
@@ -0,0 +1,31 @@
+- className: cohort
```

Comment:
There are no templates for the objects, right? They are all generated in code?

Reply:
In short yes, we can extend the schema for this file if needed, but to keep it simple for now is better to just "hardcode" the genaration of namespace, LQs, ResourceFlavor ....